notebooks/tel_and_site/subsys_req_ver/m1m3/LVV-T232-analysis_actuators.ipynb

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    %mkdir -p {dirName}
</pre><p>Instead of calling the terminal to create a folder, maybe use this <code>os.makedirs("plots", exist_ok=True)</code>? This will create a new directory in the local folder and it is more straightforward to execute and we deatach the code from your current path (<code>scratch/cslage/m1m3movies/</code>).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Done